sql/hive-thriftserver/pom.xml

```diff
@@ -41,11 +41,8 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-core_${scala.binary.version}</artifactId>
-      <version>${project.version}</version>
-      <type>test-jar</type>
-      <scope>test</scope>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
```

Comment:
@srowen I am still trying to hunt this one down - but what I observed is that mvn install of master goes through fine (between each runs, I remove $HOME/.m2 to avoid issues).
It is only this which is causing issues - will continue to hunt to see if I can resolve it; thanks for the link.